venues/MIDL.amsterdam/2018/Abstract/python/config.py

```diff
@@ -16,16 +16,15 @@
     --> my.conference/2017/Program_Chairs
 
 """
-def timestamp_GMT(year, month, day, hour=0, minute=0, second=0):
-    return int((datetime.datetime(year, month, day, hour, minute, second) - datetime.datetime(1970, 1, 1)).total_seconds() * 1000)
 
 CONFERENCE_ID = 'MIDL.amsterdam/2018/Abstract'
 ADMIN = CONFERENCE_ID + '/Admin'
 PROGRAM_CHAIRS = CONFERENCE_ID + '/Program_Chairs'
 AREA_CHAIRS = CONFERENCE_ID + '/Area_Chairs'
 REVIEWERS = CONFERENCE_ID + '/Reviewers'
-SUBMISSION_TIMESTAMP = 1523483999000
-REVIEW_TIMESTAMP = timestamp_GMT(2018,5, 9, 23, 0, 0)
+S# midnight CET = 11pm GMT
```

Comment:
this must be a typo?

make sure that you're testing this stuff before you commit